### PR TITLE
As/tap10 login and signup pages

### PR DESCRIPTION
**Pull request by 8adiq**
Done with validation for signup

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,21 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import {BrowserRouter as Router,Routes,Route} from  'react-router-dom'; 
+import SingUp from './pages/SingUp';
```
Comment (a11rew) [inline src/App.tsx]: Could you fix this typo?

**Review by a11rew — CHANGES_REQUESTED** [review]

File: src/components/SignUpForm.tsx [hunk 2]
```diff
@@ -1,39 +1,54 @@
...
-       })
-       
-       const [error,setError] = useState('')
-       const [showPassword,setShowPassword] = useState(false)
-       const [showConfirmPassword,setShowConfirmPassword] = useState(false)
+  interface formData {
+    name: string,
+    email: string,
+    password: string,
+    confirmPassword: string
```
Comment (PeterKwesiAnsah) [inline src/components/SignUpForm.tsx]: types/interfaces are global scoped

File: src/components/LogInForm.tsx [hunk 3]
```diff
@@ -1,52 +1,53 @@
...
 
 const LogInForm = () => {
-    const [{email,password},setCredentials] = useState({
-        email:'',
-        password:''
-    })
+    interface loginData {
+        email: string,
+        password: string,
+      }
```
Comment (PeterKwesiAnsah) [inline src/components/LogInForm.tsx]: Types global scoped..

File: src/components/SignUpForm.tsx [hunk 4]
```diff
@@ -42,54 +57,54 @@ const SignUpForm = () => {
...
+          name='confirmPassword'
           type={showConfirmPassword ? 'text' : 'password'}
-          value={confirmPassword}
-          placeholder='Confirm Password'
-          className='border-b-2 outline-none P-2'
-          onChange={(e) => setDetails({ password,name,email,confirmPassword: e.target.value })}
-          
+          placeholder='Confirm password'
+          className='border-b-2 outline-none P-2'   
           />
```
Comment (PeterKwesiAnsah) [inline src/components/SignUpForm.tsx:93]: it seems you are not showing validation messages.....one way of doing that..is simply conditionally rendering a message
if password && confirm password are not the same

**Review by PeterKwesiAnsah — APPROVED** [review]

**Review by PeterKwesiAnsah — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (8adiq) [reply src/App.tsx]: I fixed it.

File: src/App.tsx [hunk 5]
```diff
@@ -1,26 +1,21 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="min-h-screen flex flex-col justify-center">
+      <Router>
+        <Routes>
+          <Route path='/' element={<SingUp/>}/>
```
Comment (blitzblade) [inline src/App.tsx]: So if someone is logged in and they try to hit the "/", what happens?

Comment (8adiq) [reply src/App.tsx]: The sign up page comes up.

Comment (8adiq) [reply src/App.tsx]: I am not sure I get the question

**Pull request merged**